modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaHelidonCommonCodegen.java

```diff
@@ -769,6 +769,30 @@ String chooseVersion(String requestedVersion) {
             return chooseVersion(requestedVersion, versions);
         }
 
+        /**
+         *  Returns either the best match version of, if there is none, the requested version itself to allow references to
```

Comment:
I think there's a typo here:
```suggestion
         *  Returns either the best match version or, if there is none, the requested version itself to allow references to
```

Same in line 784.

Other than that, this PR looks good to me. Thanks for the contribution!
And indeed the timed out job was green when I ran it [here](https://github.com/martin-mfg/openapi-generator/pull/53).

Reply:
Good catch, yes a typo! I've committed your suggested change. Thanks.

Reply:
@martin-mfg Ah, your suggested change addressed one but not both typos.

If I push a fix from my local branch will you be able to re-approve the PR?

Reply:
@tjquinno Yes I can re-approve in case GitHub removes my approval from before.